app/views/admin/petitions/edit.html.erb

```diff
@@ -2,7 +2,11 @@
 
   <%= render :partial => 'petition_details', :locals => { :f => f } %>
 
-  <div>
+  <div class="row">
+    <%= link_to 'Edit petition content', admin_petition_petition_details_path(@petition) %>
+  </div>
```

Comment:
Now that #145 is merged we can put this action on the show page with the other actions.